test/accountSpec.js

```diff
@@ -81,7 +82,15 @@ describe('Accounts', function () {
       yield this.request()
         .get('/accounts')
         .auth('admin', 'admin')
-        .expect([account1, account2, account3, account4, account5, account6])
+        .expect((res) => {
+          const sortedResponse = _.sortBy(res.body, (account) => {
```

Comment:
It could, but then we wouldn't be able to create that account using the RESt api. This test checks direct insertion into the db, using `dbHelper.addAccounts`, while it `should default the balance to 0` tests using the RESt api. I think it's safest to test with the RESt api, since that's the endpoint we are exposing.